### PR TITLE
Add Questions + Fix

### PR DESCRIPTION
**Pull request by Haadi-Khan**
Added about 60 more questions, found a source for 20 of yours, and fixed a question's answer

Original:
            "Pounds of skin shed by a person in his or her lifetime.": 1
Fixed:
            "Pounds of skin shed by a person in his or her lifetime.": 2
Source: https://www.sciencefocus.com/the-human-body/how-much-skin-does-a-human-shed-in-their-life/
Also you can google it

### CODE REVIEW

**Comment by Haadi-Khan** [comment]
Found another potential error:

What is the kinetic energy stored in continental drift?

The answer on the site is 2, but this is almost definitely wrong:

Earth's crust is approximately 2.8e22 kg [see paper](https://ui.adsabs.harvard.edu/abs/2007AGUFM.V33A1161P/abstract)

It moves anywhere between 1-4 inches a year. However, most locations are considerably less than 4. Let's say 1 for the most lowball estimate possible

1 inch a year is approx 8e-10 m/s. This means that using trusty 1/2 mv^2 we get back:

1.4e22 * (8e-10)^2 = 8.96e3, aka OOM = 4

Did I do my math wrong? I believe this should be the correct method, but I may be wrong.

If this is correct, make sure to change the question's answer to 4

File: question-bank.json [hunk 1]
```diff
@@ -195,13 +166,133 @@
...
     "sources": {
         "og": {
             "link": "https://github.com/EricAndrechek/FermiQuestions#changes",
             "author": "Eric Andrechek",
             "date": "February 10, 2019"
+        },
+        "nc_state_ex": {
+            "link": "https://ncscienceolympiad.ncsu.edu/wp-content/uploads/2018/08/NC-FERMI-C.pdf",
+            "author": "NC State",
+            "date": "07/29/22"
```
Comment (EricAndrechek) [inline question-bank.json]: Can we put the author as the person who sourced the content? On the website, it links to the source, says sourced by, and then the sourcers name. That way people who contribute also get some credit. Also, I had the dates as February 10, 2019 - can we continue writing them in that format for consistency's sake? Thanks!

File: question-bank.json [hunk 2]
```diff
@@ -195,13 +166,133 @@
...
+        },
+        "nc_state": {
+            "link": "https://ncscienceolympiad.ncsu.edu/wp-content/uploads/2018/08/2018-Fermi-Questions-State-C.pdf",
+            "author": "NC State",
+            "date": "07/29/22"
```
Comment (EricAndrechek) [inline question-bank.json]: Also, do you think we should change the dates on these? I know they were originally sourced in 2019, do you think we leave the date as that when originally sourced or update it to now? I don't really think it matters, I just figure we either change the date back to 2019 and put me as the author of the addition or leave the date as now and put you as the author of the addition. Your call.

**Review by EricAndrechek — CHANGES_REQUESTED** [review]

**Comment by EricAndrechek** [comment]
> Found another potential error:
> 
> What is the kinetic energy stored in continental drift?
> 
> The answer on the site is 2, but this is almost definitely wrong:
> 
> Earth's crust is approximately 2.8e22 kg [see paper](https://ui.adsabs.harvard.edu/abs/2007AGUFM.V33A1161P/abstract)
> 
> It moves anywhere between 1-4 inches a year. However, most locations are considerably less than 4. Let's say 1 for the most lowball estimate possible
> 
> 1 inch a year is approx 8e-10 m/s. This means that using trusty 1/2 mv^2 we get back:
> 
> 1.4e22 * (8e-10)^2 = 8.96e3, aka OOM = 4
> 
> Did I do my math wrong? I believe this should be the correct method, but I may be wrong.
> 
> If this is correct, make sure to change the question's answer to 4

I agree this is correct. Different websites and sources I looked at tend to have it at 3 or 4 depending on what they say for a distance of continental drift but I think you're closest at 4. Feel free to make this change as well. Thanks!

**Review by EricAndrechek — APPROVED** [review]
Thanks!

**Comment by EricAndrechek** [comment]
@all-contributors add @Haadi-Khan for bug fixes and content

**Pull request merged**